CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-kms ChangeLog
 
+## 6.0.0 - 2021-04-TBD
+
+### Changed
+- **BREAKING**: Drop support for node 10.
+- Support `ed25519-signature-2020` signature suite.
+- Remove use of `jsonld-signatures`.
+- Remove `@digitalbazaar/did-io` and use `bedrock-did-io@1.0`.
+- Remove `did-method-key`.
+- Update dependencies to latest:
+  [bedrock-did-io@1.0](https://github.com/digitalbazaar/bedrock-did-io/pull/2),
+  [webkms-switch@4.0](https://github.com/digitalbazaar/webkms-switch/pull/9).
```

Comment:
I mentioned this elsewhere, I don't thing we want to have links to PRs in changelogs.  These links **could** be to the changelog entry however.  Also, I think these should be sublists (preceded with `-`).

Reply:
Updated e68af7c